__tests__/integration/routes/+layout.test.ts

```diff
@@ -45,9 +45,9 @@ describe('get_anon_userID', () => {
 	});
 
 	it('should redirect authorized user when accessing login url ', () => {
-		const user = { name: 'Andreas' };
+		const user = { name: 'Andreas', role: "Student" };
```

Comment:
role: "King"

Reply:
i do not appreciate your patetic attempt at a clever jest